app/assets/src/components/layout/Accordion.jsx

```diff
@@ -4,23 +4,19 @@ import cx from "classnames";
 import cs from "./accordion.scss";
 
 class Accordion extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      open: this.props.open
-    };
-  }
+  state = {};
 
   onToggle = () => {
     this.setState({
-      open: !this.state.open
+      open: !this.state.open,
+      wasToggled: true
     });
   };
 
   render() {
     const { header, children, toggleable, className } = this.props;
 
-    const open = this.state.open;
+    const open = this.state.wasToggled ? this.state.open : this.props.open;
```

Comment:
still not happy with this fix but it works

Reply:
I think a cleaner approach would be to:

Add a `defaultOpen` prop that allows you to set the `open` state on the initial component mount, but gives control over the component afterwards.

In `<DiscoverySidebar>`, don't render (or mount) the Accordions until the data has been loaded. Show a loading message when the data is still being fetched. This way, you can set the `defaultOpen` prop properly once the data is loaded (since you only want the Accordion to be open if there is data is show).